cypress/e2e/supervision/finance/finance-tab.cy.js

```diff
@@ -0,0 +1,201 @@
+      cy.get('#finance-discount-list-table').should('contain.text', '01/04/2023');
+      cy.get('#finance-discount-list-table').should('contain.text', '31/03/2024');
+      cy.get('#finance-discount-list-table').should('contain.text', 'Active');
+      cy.get('.finance-discount-list-notes').should('contain.text', 'Test applying fee reduction');
+    });
+
+    it('can show annual fee information', () => {
+      let $response = [
+        {
+          "amountPaid": 0,
+          "amountOutstanding": 100,
+          "status": {
+            "handle": "OPEN",
+            "label": "Open"
+          },
+          "sopStatus": {
+            "handle": "CONFIRMED",
+            "label": "Confirmed"
+          },
+          "allowedActions": [
+            {
+              "handle": "CREDIT",
+              "label": "Add Credit"
+            },
+            {
+              "handle": "WRITE OFF",
+              "label": "Write-off"
+            }
+          ],
+          "id": 4,
+          "feeType": "AD",
+          "reference": "AD000001\/23",
+          "raisedDate": "10\/07\/2020",
+          "startDate": "10\/07\/2020",
+          "endDate": "09\/07\/2021",
+          "amount": 100,
+          "feeRanges": [
+            {
+              "supervisionLevel": {
+                "handle": "GENERAL",
+                "label": "General"
+              },
+              "fromDate": "01\/04\/2022",
+              "toDate": "31\/03\/2024",
+              "amount": 100
+            }
+          ],
+          "ledgerEntryAllocations": [
+            {
+              "allowedActions": [],
+              "id": 1,
+              "ledgerEntry": {
+                "id": 1,
+                "type": {
+                  "handle": "CARD PAYMENT",
+                  "label": "Card payment"
+                },
+                "reference": "AD000001\/23",
+                "dateTime": "2020-07-10T00:00:00+00:00",
+                "method": "CARD PAYMENT",
+                "amount": 10000,
+                "amountAllocated": 10000,
+                "amountUnallocated": 0,
+                "status": {
+                  "handle": "APPROVED",
+                  "label": "Approved"
+                },
+                "ledgerEntryAllocations": [],
+                "childLedgerEntries": [],
+                "confirmedDate": "10\/07\/2020",
+                "isFeeWaiver": false,
+                "allowedActions": []
+              },
+              "dateTime": "2020-07-10T00:00:00+00:00",
+              "amount": 10000,
+              "status": {
+                "handle": "ALLOCATED",
+                "label": "Applied"
+              },
+              "allocatedDate": "10\/07\/2020"
+            }
+          ],
+          "batchNumber": 2,
+          "confirmedDate": "10\/07\/2020"
+        }
+      ];
+      cy.get("@client").then(({ id }) => {
+        cy.intercept("GET", `/supervision-api/v1/finance/${id}/invoices`, $response)
```

Comment:
Is there a reason why you've gone with an intercept here rather than using the real API response? If the API changes and sends different information, this E2E could continue to pass and give a false positive.